keras_hub/src/layers/modeling/non_max_suppression_test.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from keras import ops
+
+from keras_hub.src.layers.modeling.non_max_suppression import NonMaxSuppression
+from keras_hub.src.tests.test_case import TestCase
+
+
+class NonMaxSupressionTest(TestCase):
+    def test_confidence_threshold(self):
```

Comment:
add the generic `self.run_layers_tests`, example : https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/layers/modeling/alibi_bias_test.py#L16